styles/main.css

```diff
@@ -207,9 +207,9 @@ main {
 }
 
 .row-span-2 {
-    grid-row: 1 / 3;
+    grid-row: span 2;
 }
 
 .col-span-2 {
-    grid-column: 1 / 3;
+    grid-column: span 2;
 }
```

Comment:
What you have here makes .row-span-2 and .col-span-2 elements ALWAYS start on line one and end on line 3.  What you want is for them to start wherever they start and span from there